src/shared/components/resources/ResourceTable.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
```

Comment:
We should convert this into functional component.   We want to avoid making new class components.  Will show you how.